modules/govuk/templates/ckan/nginx.conf.erb

```diff
@@ -71,6 +71,11 @@ location /api/3<%= path %> {
 }
 <% end %>
 
+# Rewrite publisher to organization for CKAN 2.9 in case it has been bookmarked
+location /publisher {
+  rewrite ^(.*)/publisher(/.*)?$ $1/organization$2 last;
```

Comment:
Have you tested this? I think it might create a loop. If I've understood the docs correctly, `last` means "don't process any more rewrite rules for the current URL, instead go straight to processing `location` blocks for the rewritten URL". See the first rewrite example [here](http://nginx.org/en/docs/http/ngx_http_rewrite_module.html).

I think you may need to either change `last` to `break`, or not put the rewrite rule inside `location /publisher`.

I haven't actually tested it though, so I could be wrong :)

Reply:
I've tried it and it works, I don't mind changing it to `break` instead.

Reply:
If it works then it's almost certainly fine - I must have got it wrong :)